mne/io/tests/test_raw.py

```diff
@@ -506,6 +507,37 @@ def test_time_as_index():
     assert_array_equal(new_inds, np.arange(len(raw.times)))
 
 
+@pytest.mark.parametrize('meas_date', [None, "orig"])
+def test_crop_by_annotations(meas_date):
+    """Test crop by annotations of raw."""
+    raw_fname = op.join(op.dirname(__file__), '..', '..', 'io', 'tests',
+                        'data', 'test_raw.fif')
+    raw = read_raw_fif(raw_fname)
+
+    if meas_date is None:
+        raw.set_meas_date(None)
+
+    raw = mne.io.RawArray(raw.get_data(),
+                          raw.info,
+                          first_samp=0)
```

Comment:
@larsoner if I do 
```suggestion
                          first_samp=10_000)
```

then the test fail. Do you have any explanation of why it should be like this? 🙏

Reply:
This has been fixed by #10490, feel free to proceed with your PR :)